posixlib/src/main/scala/scala/scalanative/linux/epoll.scala

```diff
@@ -41,49 +40,36 @@ object epoll {
   @name("scalanative_epolloneshot")
   def EPOLLONESHOT: CInt = extern
 
-  type epoll_data_t = CStruct1[CLongLong] // 64-bit
-  type epoll_event = CStruct2[UInt, epoll_data_t]
-
   def epoll_create(size: CInt): CInt = extern
   def epoll_create1(flags: CInt): CInt = extern
 
   def epoll_ctl(
       epfd: CInt,
       op: CInt,
       fd: CInt,
-      event: Ptr[epoll_event]
+      event: Ptr[Byte]
```

Comment:
```suggestion
      event: CVoidPtr
```
I dont actually know if this breaks linking or not, just thought that was the new idiomatic way for unknown pointer

Reply:
let me try

Reply:
done